lib/services/local/access_list.go

```diff
@@ -223,5 +224,5 @@ func (a *AccessListService) DeleteAllAccessListMembers(ctx context.Context) erro
 }
 
 func lockName(accessListName string) string {
-	return string(backend.Key("access_list", accessListName))
+	return strings.Join([]string{"access_list", accessListName}, string(backend.Separator))
```

Comment:
Revert the file? I suppose both versions rely on backend.Key internals, but I'm not seeing much point in changing it anyway.

Reply:
This was already fixed in master, it'll go away now when I rebase.

Reply:
It looks like your review was racy here. I pushed a rebase which removed it before you commented :)